src/library_syscall.js

```diff
@@ -1118,7 +1118,10 @@ var SyscallsLibrary = {
         return 0;
       }
       case {{{ cDefine('F_GETLK') }}}:
-      case {{{ cDefine('F_GETLK64') }}}: {
+      /* case {{{ cDefine('F_GETLK64') }}}: Currently in musl F_GETLK64 has same value as F_GETLK, but if that changes, uncomment this */ {
```

Comment:
is the issue that closure would error on a duplicate key? please add that to the comment

Reply:
It issues a warning, but we don't print out closure warnings, so it got lost.